### DISCUSSION TITLE
How to set a PDF cover?

### DISCUSSION THREAD

**Discussion by Sungq1990** [gotenberg/gotenberg#834] [Q&A]
I want to convert HTML to PDF,this is my code,I want to exclude the header and footer from the cover page.

`
        $request = Gotenberg::chromium('http://localhost:3000')
            ->pdf()
            ->skipNetworkIdleEvent()
            ->outputFilename('test')
            ->header(Stream::path('/mnt/c/Users/user/Desktop/header.html'))
            ->margins(0.7, 0.3, 0, 0)
            ->footer(Stream::path('/mnt/c/Users/user/Desktop/footer.html'))
            ->html(Stream::path('/mnt/c/Users/user/Desktop/index1.html'));

        Gotenberg::save($request, '/mnt/c/Users/user/Desktop/');
`

**Comment by gulien** [comment]
Header and footer applies to all pages, and I cannot change this because it’s how Chromium is handling this feature. Only solution is to generate your cover independently and then merge the two PDF.

**Reply by Sungq1990** [reply]
I got it, thank you.
